### PR TITLE
Updating `firebase-perf` external documentation as part of Open Sourcing

### PR DESCRIPTION
**Pull request by ramanpreetSinghKhinda**
Preparing `firebase-perf` to be OSS ready. 
  - Updating main README to include `firebase-perf`
  - Updating [firebaseopensource](https://firebaseopensource.com/projects/firebase/firebase-android-sdk/) to include `firebase-perf`

**Note:** Merge only after `firebase-perf` have been Open Sourced.

### CODE REVIEW

**Review by rlazo — APPROVED** [review]

**Comment by ramanpreetSinghKhinda** [comment]
@rlazo Are there any other public documentation updates required for `fireperf`?

**Review by jeremyjiang-dev — COMMENTED** [review]
Do we need to update https://github.com/firebase/firebase-android-sdk/blob/master/.opensource/project.json and https://github.com/firebase/firebase-android-sdk/tree/master/docs

**Comment by rlazo** [comment]
> @rlazo Are there any other public documentation updates required for `fireperf`?

The only "public" doc that refers to the actual contents of the repo is the README.md So if that's updated, you're good to go.

**Comment by ramanpreetSinghKhinda** [comment]
> Do we need to update https://github.com/firebase/firebase-android-sdk/blob/master/.opensource/project.json and https://github.com/firebase/firebase-android-sdk/tree/master/docs

Good point Jeremy. Let me update this in this PR.

**Comment by rlazo** [comment]
> Do we need to update https://github.com/firebase/firebase-android-sdk/blob/master/.opensource/project.json and https://github.com/firebase/firebase-android-sdk/tree/master/docs

That's a great question. If anything needs to be changed there is **not** a blocker for OSS. It would be more an oversight from our (Core) part. I'll double check about those files.

**Review by rlazo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ dependencies:
   * `firebase-inappmessaging-ktx`
   * `firebase-inappmessaging-display`
   * `firebase-inappmessaging-display-ktx`
+  * `firebase-perf`
```
Comment (rachelsaunders) [inline README.md:25]:
```suggestion
  * `firebase-perf`
  * `firebase-perf-ktx`
```

**Review by rachelsaunders — DISMISSED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ This repository includes the following Firebase SDKs for Android:
   * `firebase-firestore`
   * `firebase-functions`
   * `firebase-inappmessaging-display`
+  * `firebase-perf`
```
Comment (jeremyjiang-dev) [inline docs/README.md:15]: Do you wanna add another `firebase-perf` to the ktx section below?

Comment (jeremyjiang-dev) [reply docs/README.md:15]: Never mind. Just saw the comments that we don't need it.

**Review by jeremyjiang-dev — APPROVED** [review]

**Review by visumickey — APPROVED** [review]

**Comment by ramanpreetSinghKhinda** [comment]
@rlazo Looks like there are some tests which has been failing. I wonder why README changes are triggering tests for those SDKs in the first place.

Also how can we avoid getting blocked on these unrelated transient errors?

**Comment by jeremyjiang-dev** [comment]
I wonder what is the best approach here for some unrelated flaky tests that are failing:
1. Should we ask people to fix them?
2. Should we merge without that test passing if we know that test is flaky by running it locally.

I think rerunning the tests may not help to solve the problem fundamentally.

**Comment by ramanpreetSinghKhinda** [comment]
> I wonder what is the best approach here for some unrelated flaky tests that are failing:
> 
> 1. Should we ask people to fix them?
> 2. Should we merge without that test passing if we know that test is flaky by running it locally.
> 
> I think rerunning the tests may not help to solve the problem fundamentally.

cc @vkryachko @rlazo

**Pull request merged**